packages/mdc-button/mdc-button.scss

```diff
@@ -86,14 +87,9 @@
 }
 
 .mdc-button--raised {
+  @include mdc-button-contained();
```

Comment:
Including `mdc-button-contained` separately here and below leads to a lot of duplicated CSS in the output that I think we can avoid. What about the following:

```scss
.mdc-button--raised,
.mdc-button--unelevated {
  @include mdc-button-contained();
}

.mdc-button--raised {
  @include mdc-elevation(2);
  ...
}
```